### PR TITLE
Tune-up the README

### PR DESCRIPTION
**Pull request by dhh**
Make it fit with the other Solid options, and explain a few more things in further detail.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,18 @@
...
 
-```ruby
-gem "solid_cable"
-```
-
-And then execute:
-```bash
-$ bundle
-```
+Solid Cable is configured by default in new Rails 8 applications. But if you're running an earlier version, you can add it manually following these steps:
```
Comment (npezza93) [inline README.md:8]: @dhh I think this might be some copy-pasta unless it is going to be the default in 8.

**Review by npezza93 — COMMENTED** [review]
Thanks for this!

File: README.md (hunk 1 above)
Comment (dhh) [reply README.md:8]: I'm very much intending that we aim for Rails 8 by default 😄. Can you hit me up on david@hey.com? Tried emailing pezza@hey.com.

Comment (npezza93) [reply README.md:8]: Whoops! That's what i get for not checking my email today! Just replied!

**Review by npezza93 — APPROVED** [review]

**Pull request merged**